src/nls/en/shortcuts-mac.json

```diff
@@ -32,7 +32,8 @@
         },
         "LAYER": {
             "CREATE_ARTBOARD": "b",
-            "DUPLICATE": "j"
+            "DUPLICATE": "j",
+            "NEW_VECTOR_MASK_FROM_SHAPE": "m"
```

Comment:
So then let's go with Ctrl-M unless, for some reason, that doesn't work in Design Space. If it doesn't, then that could conceivably be a plugin bug, since apparently it does work in standard PS.